source/guide_ackee.rst

```diff
@@ -34,47 +32,81 @@ Ackee_ is released under the `MIT License`_.
+Installation
+============
+
+Install ``yarn`` globally:
 
 .. code-block:: console
 
   [isabell@stardust ~]$ npm install yarn -g
-  /home/isabell/bin/yarn -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  /home/isabell/bin/yarnpkg -> /home/isabell/lib/node_modules/yarn/bin/yarn.js
-  + yarn@1.22.4
-  installed 1 package in 1.002s
   [isabell@stardust ~]$ 
 
-Installation
-============
+Clone the `GitHub repository <https://github.com/electerious/Ackee>`_ and install the dependencies:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee
+  [isabell@stardust ~]$ cd Ackee
+  [isabell@stardust Ackee]$ yarn
+
+Configuration
+=============
+
+Ackee Config
+------------
 
-Clone the `GitHub repository of Ackee`_:
+Create a new, empty ``.env`` file in the root of the Ackee_ directory. Add the following content, replacing the placeholders. The environment variables ``ACKEE_USERNAME`` and ``ACKEE_PASSWORD`` define the credentials to log into the web interface. If you want to use Ackee_ to analyze a domain other than ``isabell.uber.space``, you have to add it under ``ACKEE_ALLOW_ORIGIN`` (read more about the CORS headers `in the Ackee documentation <https://github.com/electerious/Ackee/blob/master/docs/CORS%20headers.md>`_). 
 
 .. code-block:: console
 
-  [isabell@stardust ~]$ git clone https://github.com/electerious/Ackee ~/ackee
-  Cloning into '/home/isabell/ackee'...
-  remote: Enumerating objects: 1528, done.
-  remote: Counting objects: 100% (1528/1528), done.
-  remote: Compressing objects: 100% (884/884), done.
-  remote: Total 7991 (delta 1016), reused 1083 (delta 641), pack-reused 6463
-  Receiving objects: 100% (7991/7991), 1.97 MiB | 3.95 MiB/s, done.
-  Resolving deltas: 100% (5471/5471), done.
-  [isabell@stardust ~]$ 
+  ACKEE_MONGODB=mongodb://<admin>_mongoroot:<password>@localhost:27017/admin
+  ACKEE_USERNAME=<ackee_username>
+  ACKEE_PASSWORD=<ackee_password>
+  ACKEE_ALLOW_ORIGIN="https://my-domain-other-than-isabell-on-stardust.de"
+
+
+Web Backend Config
+------------------
+
+.. note::
+
+    Ackee_ is running on port 3000.
+
+.. include:: includes/web-backend.rst
+
+Supervisord Daemon Setup
+------------------------
+
+Create ``~/etc/services.d/ackee.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:ackee]
+  directory=%(ENV_HOME)s/Ackee
+  command=yarn start
+  autostart=yes
+  autorestart=yes
+
+.. include:: includes/supervisord.rst
+
+Finishing installation
+======================
+
+Go to ``https://isabell.uber.space``. Login using the credentials defined in section `Ackee Config`. Add the domains you want to analyze in the web interface: After adding, it shows a tracking snippet you can add to these pages. If data does not show up after implementing the tracking script, remember that cross site tracking always takes extra consideration of `CORS <https://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`_.
```

Comment:
I wouldn't specify the domain here. The user decided that while setting up the Web backend.

Reply:
The default for projects is usually `/`. This is why this route should be correct.

Reply:
That's not what I meant. I wouldn't specify the *domain* here, also not the route. Users may see that as a general link under which all Ackee instances on the world can be found etc.

Reply:
If I understand you correctly, you would remove:

> Go to ``https://isabell.uber.space``.

I disagree on this, since the guide sets up Ackee on the root, so this link leads to the web interface. As far as I understand it, setting up a system on root is the baseline the guides.

Reply:
Seems like there is no correct behaviour there. Either a placeholder domain is uaed, but not as an anchor tag, or isabell.uber.space is used as an example, but with the indication to please use your specified domain.